elyra/cli/pipeline_app.py

```diff
@@ -149,6 +151,42 @@ def _preprocess_pipeline(pipeline_path: str, runtime: str, runtime_config: str)
     return pipeline_definition
 
 
+def _print_issues(issues):
+    # print validation issues
+    for issue in issues:
+        if issue.get('severity') == ValidationSeverity.Error:
+            click.echo(
+                f'- (Fatal error on node \'{issue["data"]["nodeName"]}\' property \'{issue["data"]["propertyName"]}\') '
+                f'- {issue["message"]}')
+        else:
+            # TODO check warning nodeNames are empty
+            click.echo(
+                f'- (Warning) - {issue["message"]}')
```

Comment:
do we want to add any additional information (minimal..like just node nodename) to warnings as well to help identify the node?

Reply:
nm, see #1999

Reply:
The reason for it not being there is #1999 but for now, I would consider this as nice to have and we can fix it on the next patch release.